### PR TITLE
feat(docker): multiplatforms builds v2

### PR DESCRIPTION
**Pull request by smerle33**
POC

### CODE REVIEW

File: resources/io/jenkins/infra/docker/Makefile [hunk 1]
```diff
@@ -34,9 +34,9 @@ lint: ## Lint the $(IMAGE_DOCKERFILE) content
 
 build: ## Build the Docker Image $(IMAGE_NAME) from $(IMAGE_DOCKERFILE)
 	@echo "== Building $(IMAGE_NAME) from $(call FixPath,$(IMAGE_DOCKERFILE)) with docker:"
-	docker buildx create --use
-    docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
-	docker buildx build \
+		docker buildx create --use
```
Comment (dduportal) [inline resources/io/jenkins/infra/docker/Makefile:37]: Could you give a name to this "builder instance" and only create it if not existing?

Also, can you remove it after the `docker run` is finished with success (no need to take care of failures: that's what the condition would be there for)

Comment (smerle33) [reply resources/io/jenkins/infra/docker/Makefile]: added in https://github.com/jenkins-infra/pipeline-library/pull/724/commits/75be361a5695998ba0f4f9e66d2da7c949d1c483

Comment (smerle33) [reply resources/io/jenkins/infra/docker/Makefile]: removed as per rubber duck

**Comment by smerle33** [comment]
close as per `ERROR: docker exporter does not currently support exporting manifest lists`

**Pull request closed without merging**